.gitignore

```
@@ -18,6 +18,8 @@
 .env.test.local
 .env.production.local
 
+.vscode
```

This ignores subdirectories too
```suggestion
.vscode/
```